app/about/page.tsx

```diff
@@ -1,3 +1,9 @@
+import { Opportunities } from "@/components/Opportunities"
+
 export default function About() {
-  return <div>About</div>
+  return (
+    <div>
+      <Opportunities />
+    </div>
```

Comment:
Do you need the `<div />`?
```suggestion
    <Opportunities />
```